db/migrate/20150623184311_add_null_constraint_to_user_fields.rb

```diff
@@ -0,0 +1,7 @@
+class AddNullConstraintToUserFields < ActiveRecord::Migration
+  def change
+    change_column_null :users, :username, false
+    change_column_null :users, :email, false
+    change_column_null :users, :password_digest, false
+  end
+end
```

Comment:
You don't need this. The null constraints have already been specified in the [migration that creates the users table](https://github.com/athifw/haxxor/blob/forgot-pass/db/migrate/20150623183027_create_users.rb#L4-L6).